docs/_plugins/search-popups/leaflet-popupmovable.md

```diff
@@ -0,0 +1,12 @@
+---
+name: Leaflet.PopupMovable
+category: search-popups
+repo: https://github.com/wrwrh/leaflet-popupmovable
+author: Yasuhiro Suzuki
+author-url: https://github.com/wrwrh/
+demo: https://wrwrh.github.io/leaflet-popupmovable/Demo/index.html
+compatible-v0:
```

Comment:
```suggestion
compatible-v0: false
```

Reply:
Thank you for the postscript.
I forgot about that test item.